doozer/doozerlib/build_info.py

```diff
@@ -445,7 +445,7 @@ def get_rhel_base_version(self) -> Optional[int]:
             if el_ver:
                 return el_ver
 
-        raise None
+        return None
```

Comment:
This is a behavioral change, as we would be raising an error (`raise None` results in a TypeError exception) instead of returning None, which should be handled by the caller. Unless we are handling this, I'd rather `raise ValueError` or the like.

Reply:
/lgtm
yet
/hold
until we figure this out

Reply:
raising ValueError now